### DISCUSSION TITLE
smtp server returns 530 when receiving emails from google

### DISCUSSION THREAD

**Discussion by ghevge** [postalserver/postal#2578] [Help with using Postal]
I've set up portal on my system, configured the DNS but when I tried to send the first email from a gmail account, I've got a delivery faild message from google with the server response status 530. Looting at the postal logs, I'm seeing the entries below. Any idea how I can get rid of this error ?

Thanks

```
smtp_1      | [9] [2023-08-11T21:16:19.914] DEBUG -- : [BDTQOB] => 250-My capabilities are
smtp_1      | [9] [2023-08-11T21:16:19.914] DEBUG -- : [BDTQOB] => 250 AUTH CRAM-MD5 PLAIN LOGIN
smtp_1      | [9] [2023-08-11T21:16:20.121] DEBUG -- : [BDTQOB] <= MAIL FROM:<myemail@gmail.com>
smtp_1      | [9] [2023-08-11T21:16:20.121] DEBUG -- : [BDTQOB] => 250 OK
smtp_1      | [9] [2023-08-11T21:16:20.223] DEBUG -- : [BDTQOB] <= RCPT TO:<admin@mydomain.com>
smtp_1      | [9] [2023-08-11T21:16:20.228] DEBUG -- : [BDTQOB] => 530 Authentication required
smtp_1      | [9] [2023-08-11T21:16:20.348] DEBUG -- : [BDTQOB] <= QUIT
smtp_1      | [9] [2023-08-11T21:16:20.348] DEBUG -- : [BDTQOB] => 221 Closing Connection
smtp_1      | [9] [2023-08-11T21:16:20.348] DEBUG -- : [BDTQOB]    Connection closed
```

**Comment by willpower232** [comment]
This means that Postal has checked for mydomain.com and doesn't know anything about it.

Postal thinks you are asking to send an email to an external email server for mydomain.com and that requires authentication to occur before sending an email.

**Reply by PrabhuShanmughapriyan** [reply]
how to fix this

**Reply by jshapiro2** [reply]
You have to setup a route for the email address you are sending to.  For example postmaster@mail.postal.com.
